pipeline.go

```diff
@@ -0,0 +1,133 @@
+package pipeline
+
+import (
+	"errors"
+	"fmt"
+	"golang.org/x/net/context"
+	"google.golang.org/appengine/datastore"
+	"google.golang.org/appengine/log"
+)
+
+// Status constants
+type Status int
+
+const (
+	initialized Status = 0
+	broken             = 1
+	building           = 2
+	opened             = 3
+	resizing           = 4
+	updating           = 5
+	recreating         = 6
+	closing            = 8
+	closed             = 9
```

Comment:
@minimum2scp I fixed the definitions of them.
https://github.com/groovenauts/blocks-concurrent-batch-agent/pull/1/commits/f68624f7b561a9f14c291ea114569e7c85d96a14
Thank you!